src/components/Company/Offers/Manage/CompanyOffersManagementWidget.spec.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { screen } from "@testing-library/react";
+
+import CompanyOffersManagementWidget from "./CompanyOffersManagementWidget";
+import { renderWithStore } from "../../../../test-utils";
+import { MuiPickersUtilsProvider } from "@material-ui/pickers";
+import DateFnsUtils from "@date-io/date-fns";
+
+
+describe("App", () => {
+    test("Renders Component", async () => {
+        renderWithStore(
```

Comment:
you should wrap this in an act() call, since the render of the component itself will cause a self-update (due to the fetch of offers). In order to test correctly the different states, you should mock the service you are using so that it does not return (to test loading) in a test case, and then another where the service returns something

Reply:
I've wrapped it around the act(). However, I wasn't able to mock the service, since on this branch the service itself is returning a MOCK_OFFERS, so it doesn't use fetch. 
I'm not sure if that is the only way, but from what I searched, to mock the service I should mock the fetch call. So I left a comment and maybe I complete the test on the branch where the service doesn't return a mock?